src/matrix/matrixfdist.jl

```diff
@@ -95,6 +95,25 @@ end
 
 @inline partype(d::MatrixFDist{T}) where {T <: Real} = T
 
+#  Konno (1988 JJSS) Corollary 2.4.i
+function cov(d::MatrixFDist, i::Int, j::Int, k::Int, l::Int)
+    p = dim(d)
+    n1, n2, PDB = params(d)
+    n2 > p + 3 || throw(ArgumentError("cov only defined for df2 > dim + 3"))
+    n = n1 + n2
+    B = Matrix(PDB)
+    n1 * (n - p - 1) * inv((n2-p) * (n2-p-1) * (n2-p-3)) * (2inv(n2-p-1) * B[i,j] * B[k,l] + B[j,l] * B[i,k] + B[i,l] * B[k,j])
```

Comment:
```suggestion
    n1*(n - p - 1)*inv((n2 - p)*(n2 - p - 1)*(n2 - p - 3))*(2inv(n2 - p - 1)*B[i,j]*B[k,l] + B[j,l]*B[i,k] + B[i,l]*B[k,j])
```
Nicer to align precedence of `*` vs `+` with the typography/spaces here